doc/source/aggregation_with_dissolve.rst

```diff
@@ -57,3 +57,14 @@ If we are interested in aggregate populations, however, we can pass different fu
 
 .. toctree::
    :maxdepth: 2
+
+### Dissolve Arguments
+
+The ``aggfun =`` argument defaults to 'first' which means that the first row of attributes values found in the dissolve routine will be applied to the resultant dissolved geodataframe.
```

Comment:
aggfun -> aggfunc

Maybe "applied" -> "assigned" ? 
It sounds a bit strange to "apply values"

Reply:
assigned is much better. thank you @jorisvandenbossche